lib/filter/filter-expr-grammar.ym

```diff
@@ -168,6 +162,16 @@ filter_plugin
           }
 	;
 
+filter_identifier
+  : LL_IDENTIFIER
+  | KW_THROTTLE
+      {
+        msg_warning_once("WARNING: throttle() is deprecated, the filter has been renamed to rate-limit()",
```

Comment:
WARNING: the throttle() filter has been renamed to rate-limit() in syslog-ng VERSION, please update your configuration to use the name rate-limit() instead of throttle()